setup.py

```diff
@@ -45,6 +45,7 @@
         "snakemake>=7.22.0",
         "pyyaml>=6.0",
         "matplotlib>=3.5.3",
+        "importlib_resources",
```

Comment:
This is unnecessary. The `importlib.resources` module is part of the Python standard library starting with version 3.7. The `importlib_resources` package is only required for compatibility with older versions of Python.

Reply:
I had to conda install it into my current environment with Python 3.10.

Reply:
Everywhere you have `import importlib_resources` you should replace with `import importlib.resources`. I missed that in my review.

Reply:
ahh okay! I will fix that.

Reply:
All fixed.